.github/workflows/scripts-setuptools-head.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13']
+        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
```

Comment:
Here 3.8 did not get restored.

Reply:
Correct: this uses setuptools HEAD, so it will fail with 3.8.